channel/consensus_channel/consensus_channel.go

```diff
@@ -639,9 +639,9 @@ func (p *Proposal) Target() types.Destination {
 
 // SignedProposal is a Proposal with a signature on it.
 type SignedProposal struct {
-	state.Signature
-	Proposal Proposal
-	TurnNum  uint64
+	Signature state.Signature
```

Comment:
I wasn't sure how to handle serializing this when the signature was an embedded struct, so I pulled it out into a `Signature` field.